posttroll/listener.py

```diff
@@ -81,17 +80,13 @@ def stop(self):
 
 
 class Listener(object):
-
-    '''PyTroll listener class for reading messages for eg. operational
-    product generation.
-    '''
+    """PyTroll listener class for reading messages for eg. operational product generation."""
 
     logger = logging.getLogger("Listener")
 
     def __init__(self, topics=None, queue=None, addresses=None,
                  nameserver="localhost", services=""):
-        '''Init Listener object
-        '''
+        """Initialize Listener object."""
```

Comment:
Should there be a possibility to pass the config dictionary to the constructor too?

Reply:
Maybe some day. At this stage I just wanted to change the `subscriber` to be dict-configurable so that `ListenerContainer` could be used without a nameserver. It would also add yet another `kwarg` if we want to keep backwards compatibility.